src/components/accounts/accounts.html

```diff
@@ -1,7 +1,15 @@
-<div class="accounts">
```

Comment:
you deleted start of `<div class="accounts">`
I mounted account dialog here. If you don't want this `<div>` remove the end `</div>`.
And in this case: is it `ok` that account dialog will be outside of accounts component?